test/Datadog.Trace.ClrProfiler.IntegrationTests/Elasticsearch5Tests.cs

```diff
@@ -4,6 +4,8 @@
 using Xunit;
 using Xunit.Abstractions;
 
+#if !NET452
```

Comment:
Was there an issue on `net452`?

Reply:
It failed for me locally and I had added it. I noiticed the elasticsearch 5 sample doesn't run for 452.
![image](https://user-images.githubusercontent.com/1801443/58966355-2a7e6180-8780-11e9-80c2-22f7e762f2d6.png)

Reply:
Yup, you're right! I wonder why this wasn't caught before.

Reply:
Just another reason to get windows integration tests in CI again

Reply:
Soon!